mucommander-viewer-image/build.gradle

```diff
@@ -6,17 +6,35 @@ dependencies {
     api project(':mucommander-viewer-api')
     api project(':mucommander-translator')
 
+    comprise 'com.twelvemonkeys.imageio:imageio-core:3.11.0'
+
+    // image plugins to handle additional types (like psd) or faster alternatives to built-in ones (jpeg)
+    comprise 'com.twelvemonkeys.imageio:imageio-jpeg:3.11.0'
+    comprise 'com.twelvemonkeys.imageio:imageio-tiff:3.11.0'
+    comprise 'com.twelvemonkeys.imageio:imageio-psd:3.11.0'
+    comprise 'com.twelvemonkeys.imageio:imageio-webp:3.11.0'
+    // requirements for plugins
+    comprise 'com.twelvemonkeys.common:common-lang:3.11.0'
+    comprise 'com.twelvemonkeys.common:common-io:3.11.0'
+    comprise 'com.twelvemonkeys.imageio:imageio-metadata:3.11.0'
```

Comment:
I guess few of these libs are retrieved for the pdf bundle as well, right? I wonder if we can introduce it as a separate bundle(s)

Reply:
Yes, a separate bundle grouping Twelve Monkey would be nice. But, I'm not sure if I have enough nerves to do that :)